doc/sphinx/damage/larsonmiller.rst

```diff
@@ -0,0 +1,44 @@
+Larson Miller Damage
+====================
+
+Overview
+--------
+
+This model implements the damage model
+
+.. math::
+   \omega_{n+1} = \omega{n} + \frac{\Delta t_{n+1}}{t_R\left(\sigma_{eff} \left(1-\omega_{n+1}\right) \right)}
+
+where :math:`\sigma_{eff}` is a modular effective stress, defined by a :ref:`effective-stress` object, and :math:`t_R` is a time-to-rupture Larson-Miller relation provided
+by a :ref:`larson-miller` object.
+
+This model implements classical time-fraction damage summation in a finite 
+element framework.  It currently does not include graceful element deletion
+routines, though those may be added in the future.  This means the simulation
+can only progress up to the point of damage nucleation and not beyond.
```

Comment:
damage nucleation? isn't that the first time instance in which `omega>0`
I would rephrase saying that the simulation in NEML will not be able to reach `omega=1` because NEML doesn't have element deletion procedure.

Reply:
Also, please add a reference to the Modular creep damage page for the use of ekill dkill and sfact

Reply:
Sure.